test/unit/math/expect_near_rel.hpp

```diff
@@ -87,7 +87,7 @@ void expect_near_rel_finite(const std::string& msg, const std::vector<T1>& x1,
  * @param x2 second argument to test
  * @param tol relative tolerance
  */
-template <typename T1, typename T2>
+template <typename T1, typename T2, require_all_stan_scalar_t<T1, T2>...>
```

Comment:
Both of these two functions accept scalars. Why do they have different requires?

Reply:
The other one also accepts vectors

Reply:
Oh woah I totally missed the vector version below, idk how that happened. That's why those were different but yeah that could just be scalar